### PR TITLE
i18n: developer documentation

### PR DESCRIPTION
**Pull request by dachary**
## Status

Ready for review

## Description of Changes

Developer oriented documentation for i18n.

## Deployment

Only documentation, no deployment side effect.

### If you made changes to documentation:

- [x] Doc linting passed locally

### CODE REVIEW

File: docs/development/i18n.rst [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+The `pybabel`_ command line is wrapped into the ``manage.py
+translate`` helper for convenience. It is designed to be used by
+developers, to run tests with fixtures and during packaging.
+
+Updating strings to be translated
+---------------------------------
+
+After modifying a string in the code, templates or javascript, the
+``securedrop/translations/messages.pot`` file must be updated by
+running the following command at the root of the git tree, in the
```
Comment (redshiftzero) [inline docs/development/i18n.rst]: "at the root of the git tree" -> in `/vagrant/securedrop`

File: docs/development/i18n.rst [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+reviewed and commited. Once merged in develop, the changes will be
+visible in the `Weblate`_ web interface used by translators because it
+watches the develop branch of the SecureDrop repository.
+
+Compiling translations
+----------------------
+
+The `gettext`_ needs a compiled file for each language (the ``*.mo``
+files). This can be done by running the following command at the root
+of the git tree, in the development virtual machine::
```
Comment (redshiftzero) [inline docs/development/i18n.rst]: "at the root of the git tree" -> in `/vagrant/securedrop`

File: docs/development/i18n.rst [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+
+The updated ``securedrop/translations/messages.pot`` should then be
+reviewed and commited. Once merged in develop, the changes will be
+visible in the `Weblate`_ web interface used by translators because it
+watches the develop branch of the SecureDrop repository.
+
+Compiling translations
+----------------------
+
+The `gettext`_ needs a compiled file for each language (the ``*.mo``
```
Comment (redshiftzero) [inline docs/development/i18n.rst]: "The `gettext`_ needs" -> "`gettext`_ needs"

File: docs/development/i18n.rst (hunk 1 above)
Comment (dachary) [inline docs/development/i18n.rst]: done

File: docs/development/i18n.rst (hunk 3 above)
Comment (dachary) [inline docs/development/i18n.rst]: done

File: docs/development/i18n.rst (hunk 2 above)
Comment (dachary) [inline docs/development/i18n.rst]: done

**Comment by dachary** [comment]
@redshiftzero thanks for the careful review :-) Modified & repushed.

**Pull request merged**